src/modals/join-airdrop/index.tsx

```diff
@@ -86,88 +66,96 @@ const Component: FC = () => {
-          )}
-        />
-        <Button htmlType="submit" style={{ display: "none" }} />
-      </Form>
-
-      <div className="hint">
-        <span className="desc">
-          You are registering your Public Keys and vault addresses.
-        </span>
-        <a
-          href="https://github.com/vultisig/airdrop-registry"
-          rel="noopener noreferrer"
-          target="_blank"
-          className="link"
-        >
-          Inspect the code here.
-        </a>
-      </div>
-
-      <div className="hint">
-        <span className="desc">
-          Your Airdrop Share is based on how long you have kept funds in
-          Vultisig for. Only Layer1 assets and tokens on the 1inch Token List
-          apply.
-        </span>
-      </div>
-
-      <div className="hint">
-        <span className="desc">No other information is collected.</span>
-        <a
-          href="https://github.com/vultisig/docs/blob/main/other/privacy.md"
-          rel="noopener noreferrer"
-          target="_blank"
-          className="link"
-        >
-          Read the Privacy Policy here
-        </a>
-      </div>
-
-      <div className="hint">
-        <span className="desc">
-          You can register as many times as you like.
+      <span className="heading">Registred Vaults:</span>
+
+      <List
+        dataSource={vaults}
+        renderItem={(item, index) => (
+          <List.Item
+            key={index}
+            extra={
+              <Switch
+                loading={loading}
+                checked={item.joinAirdrop}
+                onClick={() => handleToggle(item)}
+              />
+            }
+          >
+            <List.Item.Meta title={item.alias} />
+          </List.Item>
+        )}
+      />
+
+      <span className="hint">
+        <Info />
+
+        <span>
+          Unregistering a vault removes it from the leaderboard. It may take up
+          to a day for the balance to be reflected again.
         </span>
-      </div>
+      </span>
+
+      <Collapse
+        items={[
+          {
+            key: "1",
+            label: "Privacy Information",
+            children: (
+              <>
+                <div className="text">
+                  <span className="desc">
+                    You are registering your Public Keys and vault addresses.
+                  </span>
+                  <a
+                    href="https://github.com/vultisig/airdrop-registry"
+                    rel="noopener noreferrer"
+                    target="_blank"
+                    className="link"
+                  >
+                    Inspect the code here.
+                  </a>
+                </div>
+
+                <div className="text">
+                  <span className="desc">
+                    Your Airdrop Share is based on how long you have kept funds
+                    in Vultisig for. Only Layer1 assets and tokens on the 1inch
+                    Token List apply.
```

Comment:
Please change the paragraph to:
`Your Airdrop Share is based on the amount of assets in Vultisig multiplied by time in Vaults.  The assets counted for the airdrop can be seen here.`

Link the `here` to: [docs](https://docs.vultisig.com/vultisig-token/airdrop#eligble-assets)

Reply:
I will get this one merged and deployed it first , please address this in a seperate PR?